vernac/declare.ml

```diff
@@ -192,35 +192,35 @@ let { Goptions.get = private_poly_univs } =
     ~value:true
     ()
 
-let universes_of_body_type body typ =
+let universes_of_body_type ~used_univs body typ =
   let used_univs_body = Vars.universes_of_constr body in
```

Comment:
```suggestion
  let used_univs_body = Vars.universes_of_constr ~init:used_univs body in
```

Reply:
I don't think this one is necessary as the used_univs are already included in used_univs_typ (below).